chemprop/args.py

```diff
@@ -8,21 +8,42 @@
 from warnings import warn
 
 import torch
-from tap import Tap  # pip install typed-argument-parser (https://github.com/swansonk14/typed-argument-parser)
+from tap import (
+    Tap,
+)  # pip install typed-argument-parser (https://github.com/swansonk14/typed-argument-parser)
```

Comment:
```suggestion
from tap import Tap
```
Tap is included in the environment construction, it doesn't need the extra comment here that is causing trouble.